src/modules/settings/index.ts

```diff
@@ -76,6 +76,7 @@ Settings.add(new BooleanSetting('showCurrencyGainedAnimation', 'Show currency ga
 Settings.add(new BooleanSetting('showCurrencyLostAnimation', 'Show currency lost animation', true));
 Settings.add(new BooleanSetting('hideChallengeRelatedModules', 'Hide challenge related modules', false));
 Settings.add(new BooleanSetting('disableRightClickMenu', 'Disable the right click menu', true));
+Settings.add(new BooleanSetting('showPokerusMapStatus', 'Include Resistant Pokerus status in Map colors', false));
```

Comment:
I don't really understand why we need to be able to disable it? It's last in the list, so if we just default it to white, it will just act as completed, until changed

Reply:
Thats a good call, I honestly didn't think of just defaulting it to white 😅

Reply:
Removed boolean setting, set new setting as white